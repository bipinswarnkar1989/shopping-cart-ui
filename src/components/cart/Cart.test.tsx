import React from 'react';
import { render } from '@testing-library/react';

import { Cart } from '.';

describe('<Cart />', () => {
    it('should render as expected', async () => {
        const { container } = render(<Cart />);
        expect(container).toMatchSnapshot();
    })
})