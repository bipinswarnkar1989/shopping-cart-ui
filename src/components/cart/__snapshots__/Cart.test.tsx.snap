// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cart /> should render as expected 1`] = `
<div>
  <div
    class="cart"
  >
    <div
      class="cart__header"
    >
      <div
        class="cart___header__title"
      >
        My Cart (
        2
        )
      </div>
    </div>
    <div
      class="cart__item"
    >
      <div
        class="cart__item__left"
      >
        <div
          class="cart__item__left__img"
        >
          <img
            alt="Motorola One Fusion+ (Twilight Blue, 128 GB)"
            src="https://rukminim1.flixcart.com/image/224/224/kb6tyfk0/mobile/d/n/z/motorola-one-fusion-pakf0002in-original-imafsh2zf4y3kayz.jpeg?q=90"
          />
        </div>
        <div
          class="carcart__item__left__quantity"
        >
          <div
            class="carcart__item__left__quantity_decrement"
          >
            <button
              class="circle_button"
              data-testid="decrementBtn"
            >
              –
            </button>
          </div>
          <div
            class="carcart__item__left__quantity_input"
          >
            <input
              data-testid="quantity"
              type="text"
              value="2"
            />
          </div>
          <div
            class="carcart__item__left__quantity_increment"
          >
            <button
              class="circle_button"
              data-testid="incrementBtn"
            >
              +
            </button>
          </div>
        </div>
      </div>
      <div
        class="cart__item__right"
      >
        <div
          class="cart__item__right_name"
        >
          Motorola One Fusion+ (Twilight Blue, 128 GB)
        </div>
        <div
          class="cart__item__right_price"
        >
          <span>
            ₹
          </span>
          34998
        </div>
        <div
          class="cart__item__right__remove"
        >
          <button
            class="cart__item__right__remove_btn"
            data-testid="removeBtn"
          >
            Remove
          </button>
        </div>
      </div>
    </div>
    <div
      class="cart__item"
    >
      <div
        class="cart__item__left"
      >
        <div
          class="cart__item__left__img"
        >
          <img
            alt="Nova Prime series NHT 1094 Runtime"
            src="https://rukminim1.flixcart.com/image/224/224/jxp08sw0/trimmer/f/c/r/prime-series-nht-1094-nova-original-imafg3hkucaydsym.jpeg?q=90"
          />
        </div>
        <div
          class="carcart__item__left__quantity"
        >
          <div
            class="carcart__item__left__quantity_decrement"
          >
            <button
              class="circle_button"
              data-testid="decrementBtn"
              disabled=""
            >
              –
            </button>
          </div>
          <div
            class="carcart__item__left__quantity_input"
          >
            <input
              data-testid="quantity"
              type="text"
              value="1"
            />
          </div>
          <div
            class="carcart__item__left__quantity_increment"
          >
            <button
              class="circle_button"
              data-testid="incrementBtn"
            >
              +
            </button>
          </div>
        </div>
      </div>
      <div
        class="cart__item__right"
      >
        <div
          class="cart__item__right_name"
        >
          Nova Prime series NHT 1094 Runtime
        </div>
        <div
          class="cart__item__right_price"
        >
          <span>
            ₹
          </span>
          799
        </div>
        <div
          class="cart__item__right__remove"
        >
          <button
            class="cart__item__right__remove_btn"
            data-testid="removeBtn"
          >
            Remove
          </button>
        </div>
      </div>
    </div>
    <div
      class="cart__total"
    >
      <div>
        Total Amount:
      </div>
      <div
        class="cart__total_amount"
      >
        <span>
          ₹
        </span>
        70795
      </div>
    </div>
  </div>
</div>
`;
